apps/web/src/components/file-browser/FileListItem.tsx

```
@@ -118,19 +146,24 @@ export function FileListItem({
+        items = [{ id: item.id, type: item.type }];
+      }
+
       e.dataTransfer.setData(
         'application/json',
-        JSON.stringify({
-          id: item.id,
-          type: item.type,
-          parentId,
-        })
+        JSON.stringify({ items, parentId })
       );
```

ESLint

[error] Replace `⏎········'application/json',⏎········JSON.stringify({·items,·parentId·})⏎······` with `'application/json',·JSON.stringify({·items,·parentId·})`

(prettier/prettier)